src/main/java/seedu/address/logic/commands/FavouriteCommand.java

```diff
@@ -18,12 +18,12 @@
     public static final String COMMAND_WORD = "favourite";
     public static final String MESSAGE_FAVOURITE_PERSON_SUCCESS =
             "Favourited Client %1$s! Open up Favourites window to see if he/she is in it!\n"
-            + "If you have your Favourites window opened already, \n"
-            + "close it and re-open it (via 'fw' command or 'Favourites' button) to refresh the data!";
+            + "If the Favourites window is already open, close it and re-open it\n"
+            + "(via 'fw' command or 'Favourites' button from the 'File' tab, or press 'F3' key) to refresh the data!";
     public static final String MESSAGE_UNFAVOURITE_PERSON_SUCCESS =
             "Unfavourited client %1$s! Check that he/she is removed from the Favourites window!\n"
-            + "If you have your Favourites window opened already, \n"
-            + "close it and re-open it (via 'fw' command or 'Favourites' button) to refresh the data!";;
+            + "If you have your Favourites window opened already, close it and re-open it\n"
+            + "(via 'fw' command or 'Favourites' button from the 'File' tab , or press 'F3' key) to refresh the data!";;
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Favourites a selected client "
```

Comment:
Minor thing, but could you change it to `": Favourite/Unfavourite a selected client"`? 

I also did the same with `RemindCommand` as the `Favourite` command can be used to favourite & unfavourite a Client, similar to how the `Remind` command can be used to add, edit or remove a `Reminder` 👍 

https://github.com/AY2122S2-CS2103-W16-4/tp/blob/08f1a3f20f4b3919c376ba4fdfa512169d4ef73b/src/main/java/seedu/address/logic/commands/RemindCommand.java#L37

Reply:
Sure thing! Thanks for the catch on unfavourited success message not being standardized along with favourite success message.